src/components/Common/AvatarEditModal.tsx

```diff
@@ -77,14 +79,40 @@ const AvatarEditModal = ({
   );
   const { t } = useTranslation();
   const [isDragging, setIsDragging] = useState(false);
+  const [crop, setCrop] = useState({ x: 0, y: 0 });
```

Comment:
Let's combine all the crop related states into one object state (all except error message)